### DISCUSSION TITLE
Кодирование полиса ОМС

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#82] [Административная РГ] [answered]
Конспект обсуждения на встрече 15.02.22:
- Указать, что есть два смысла информации: идентификатор и страховое покрытие.
- Предложения по кодированию Coverage, ранее принятые – в студию (Иванов).
- Требует принятия решения: Предложение В.Родионова: в идентификаторах пациента указывать только номер полиса, остальное (даты действия, страховую компанию) – в Coverage. М.Алексеев: в текущем API Нетрики все эти данные включены в идентификатор.
- Описать, как проводить поиск пациента по расширенным данным полиса (с учетом дат и СК), если они будут храниться в Coverage
- Тип полиса ОМС (М.Алексеев): старого образца, нового образца, временный полис.
- Предлагается держать в Identifier только актуальный номер, историю не создавать
- А.Звягин: ЕНП должен быть в Identifier, потому что это единый номер застрахованного в системе ОМС, а не полис. Полис – в Coverage.
Резюме: в Identifier пишем type - тип полиса, system value – полис и его номер.

**Comment by ai-fhir** [comment]
Предложения по кодированию Coverage, ранее принятые - не совсем то что я хотел описать, я хотел обсудить формат записи в Identifier.

Основная мысль, что все документы, имеющие серию и номер, должны кодироваться единообразно. У нас много примеров таких документов паспорт РФ, новый полис ОМС, лицензии (?). Предыдущие обсуждение для ДУЛ было [тут](https://github.com/fhir-ru/core/issues/2)

**Comment by ai-fhir** [comment]
Как проводить поиск пациента по расширенным данным полиса (с учетом дат и СК), если они будут храниться в Coverage:

## Поиск через chained параметры:

```
GET [base]/Patient?_has:Coverage:subscriber:identifier=[system]|[code]
```

Основная проблема, что сложно задать несколько условий поиска в Coverage. Можно задать несколько `_has` условий, но они будут выполняться независимо.

## Поиск через _include/_revinclude

GET [base]/Coverage?[параметры поиска]&_include=Coverage:subscriber

Честный поиск по всем полям Coverage и затем сервер возвращает не только полис, но и пациента

## Поиск через GraphQL

https://www.hl7.org/fhir/graphql.html

В теории тоже решает проблему, но у меня нет опыта использования

**Comment by eugenekogan-spb** [comment, marked as answer]
Предложения в текст IG

Полис ОМС регистрируется:
* в виде атрибута Patient.Identifier, где system = (Полис ОМС), value = Номер полиса
* в виде ресурса Coverage, где, кроме номера полиса, заполняются при необходимости даты начала и окончания действия, выдавшая страховая компания.
В случае смены страховой компании следует создать новый ресурс Coverage. При этом идентификатор пациента остается тем же, поскольку этот номер является номером застрахованного в системе ОМС.

и комментарии по поиску пациента с помощью полиса - выше в этой дискуссии

**Comment by eugenekogan-spb** [comment]
https://www.consultant.ru/law/hotdocs/12229.html/
Отсюда можно взять название системы идентификации
